core/storage/feedback/gae_models.py

```diff
@@ -88,18 +98,44 @@ def generate_new_thread_id(cls, exploration_id):
 
     @classmethod
     def generate_full_thread_id(cls, exploration_id, thread_id):
+        """Generates a full thread ID, given the exploration ID and
+        the thread ID.
+
+        Args:
+            exploration_id: str. The exploration ID the thread belongs to.
+            thread_id: str. The thread ID.
+
+        Returns:
+            str. Full thread ID.
+        """
         return '.'.join([exploration_id, thread_id])
 
     @property
     def thread_id(self):
+        """Returns the thread_id (not including the exploration_id) for this
+        model instance.
+
+        Returns:
+            str. thread_id for this FeedbackThreadModel instance.
+        """
         return self.id.split('.')[1]
 
     @classmethod
     def create(cls, exploration_id, thread_id):
         """Creates a new FeedbackThreadModel entry.
 
-        Throws an exception if a thread with the given exploration ID and thread
-        ID combination exists already.
+        Args:
+            exploration_id: str. The exploration ID the thread belongs to.
```

Comment:
Nit: The ID of the exploration the thread belongs to.

(Rationale: the thread belongs to an exploration, not to an exploration ID.)